.github/workflows/main.yml

```diff
@@ -9,19 +9,26 @@ concurrency:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
         go-version: [1.20.x, 1.21.x, 1.22.x]
         criu_branch: [master, criu-dev]
+        exclude:
+          - os: windows-latest
+            criu_branch: criu-dev
```

Comment:
Why is this excluded?

Reply:
Because on macos‑latest and windows‑latest only the CRIT build test is run. Without excluding them, the same test will be executed twice.

Reply:
Could you please add this as a comment into the file?